packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -196,21 +195,21 @@
             width: -webkit-fill-available;
         }
     }
-}
-
-.dc-input-wrapper__button,
-button.dc-input-wrapper__button {
-    top: 0.6rem;
-    z-index: auto;
-}
 
-.dc-input-field {
-    @include mobile {
-        width: 100%;
+    .dc-input-wrapper__button,
+    button.dc-input-wrapper__button {
+        top: 0.6rem;
```

Comment:
I was just checking your test link cause I thought this would misalign the button in the input field and it did 😢 

this is what it should look like (production)
<img width="339" alt="Screenshot 2023-06-08 at 9 26 39 AM" src="https://github.com/binary-com/deriv-app/assets/103412909/8a6918f8-833e-48be-b644-24cbfaa7f86e">

the button is misaligned just slightly but can be fixed easily 👍 
<img width="331" alt="Screenshot 2023-06-08 at 9 26 09 AM" src="https://github.com/binary-com/deriv-app/assets/103412909/21bb6c62-1e4e-487c-9c80-8610eff185d8">

Reply:
Thank you for noticing, it's just that I was able to find this component in Cashier🥲
Let me fix